src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,26 @@
+import { ErrorSpec } from '../../types/ErrorSpec';
+import { ErrorMessage } from '../ErrorMessage/ErrorMessage';
+
+type Props = {
+  error: ErrorSpec | null;
+  closeError: () => void;
```

Comment:
if a function is called on some event, then the prop should be named with the prefix `on`
like this `onHideError`